unit-tests/algo/thermal-loop/test-table-parsing.cpp

```diff
@@ -0,0 +1,35 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+//#cmake:add-file ../../../src/algo/thermal-loop/*.cpp
+#include "../algo-common.h"
+#include "./create-syntetic-data .h"
+#include "algo/thermal-loop/l500-thermal-loop.h"
+
+
+using namespace librealsense::algo::thermal_loop::l500;
+
```

Comment:
What about build_raw_data()? Need to test it for a default table, etc. -- and its behavior is undefined for a small/big table, no?

Reply:
added unitests for build_raw_data()